### ISSUE TITLE
Removing functions used for debugging.

### ISSUE THREAD

**Issue by Denz1994** [phetsims/unit-rates#188]
@pixelzoom Should functions used for debugging and not explicitly called be removed from the published sim or will these functions help with maintenance?

For example : common/model/rate.js

```js
 /**
     * String representation. For debugging and logging only. Do not rely on the format of this!
     * @returns {string}
     * @public
     */
    toString: function() {
      return this.numeratorProperty.value + '/' + this.denominatorProperty.value;
    }
```

**Comment by Denz1994** [comment]
rate.equals() follows the same format
```js
// @public
    equals: function( object ) {
      return ( object instanceof Rate ) &&
             ( object.numeratorProperty.value === this.numeratorProperty.value) &&
             ( object.denominatorProperty.value === this.denominatorProperty.value);
    }
```

**Comment by pixelzoom** [comment]
I've used `Rate.toString` repeatedly to inspect `Rate` instances in the console. And I feel that it will be useful for future maintenance, so it stays.

I've removed `Rate.equals`.  Was used in some early debugging, but unlikely to be useful in the future.

Btw... In general, `toString` and `equals` are common things to implement for types, whether they are used or not, and especially for debugging.

Back to @Denz1994 to verify.
